services/server/config/config_test.go

```diff
@@ -87,14 +87,17 @@ func TestResolveImports(t *testing.T) {
 		filepath.Join(tempDir, "config_*.toml"), // Glob
 		filepath.Join(tempDir, "./test.toml"),   // Path clean up
 		"current.toml",                          // Resolve current working dir
```

Comment:
The comment `Resolve current working dir` looks wrong, files with non-absolute paths are related to the parent configuration file directory, not the current working directory